code/src/App.js

```diff
@@ -1,9 +1,61 @@
-import React from 'react'
+import React, { useState, useEffect } from "react";
+
+import { MessageList } from "components/MessageList";
+import { MessageInput } from "components/MessageInput";
+
 
 export const App = () => {
+  const MESSAGES_URL = "https://happy-thoughts-technigo.herokuapp.com/thoughts";
+
+  const [messages, setMessages] = useState([]);
+
+  // denna funktion sätter igång att hämta nytt meddelande när messageinput är mounted
+  useEffect(() => {
+    fetchMessages();
+  }, []);
+
+  // fångar in array av nya meddelanden från API och uppdaterar state, res=sesultat, data=rådata från arrayen
```

Comment:
If we leave comments in our code, they should be written in english - consider it as latin for developers

Reply:
Now changed to latin....I mean english 😊